config/telink/chip-module/zephyr-util.cmake

```diff
@@ -0,0 +1,73 @@
+#
```

Comment:
It's a copy-paste of `config/nrfconnect/chip-module/zephyr-util.cmake`. Now that we will have another platform based on Zephyr it would make sense to extract some build files shared between nrfconnect and your platform to `config/zephyr` directory. This file looks like a good candidate for that. Could you move it to `config/zephyr/chip-module` and use in both platforms?

Reply:
Nice idea, I have the same thoughts that we would gather shared code in **config/zephyr** folder.
Here I see that

- module.yml
- zephyr-util.cmake
- Kconfig

Could be shared.
Did I miss something?